doc/releases/changelog-dev.md

```diff
@@ -310,7 +314,16 @@
   [(#5978)](https://github.com/PennyLaneAI/pennylane/pull/5978)
 
 * `qml.AmplitudeEmbedding` has better support for features using low precision integer data types.
-[(#5969)](https://github.com/PennyLaneAI/pennylane/pull/5969)
+  [(#5969)](https://github.com/PennyLaneAI/pennylane/pull/5969)
+
+* `qml.BasisState` now works with jax.jit.
+  [(#6021)](https://github.com/PennyLaneAI/pennylane/pull/6021)
+
+* `qml.BasisEmbedding` now gives the correct decomposition.
+  [(#6021)](https://github.com/PennyLaneAI/pennylane/pull/6021)
+
+* `qml.BasisEmbedding` now works with lightning.qubit and jax.jit.
+  [(#6021)](https://github.com/PennyLaneAI/pennylane/pull/6021)
```

Comment:
Not necessarily a blocker, but this is a potential sign that the PR is doing too much.

Reply:
I agree, however in this case it came naturally when reformatting the classes

Reply:
Agree with Tom, this PR is too long.